themes/inpycon2020/templates/base.html

```diff
@@ -10,6 +10,19 @@
     <link rel="shortcut icon" href="/blog/theme/images/favicon.png"/>
     <link rel="stylesheet" href="/blog/theme/css/bootstrap.min.css"/>
     <link rel="stylesheet" href="/blog/theme/css/styles.css"/>
+    {% if article is defined %}
+    <meta property="og:type" content="article" />
+    <meta property="og:url" content="{{ SITEURL }}/{{ article.url }}" />
+    <meta property="og:title" content="{{ article.title | replace("\"", "&quot;") }}" />
+    <meta property="og:description" content="{{ article.content | striptags | replace("\"", "&quot;") | truncate(124, False, '...') }}" />
+    <meta property="og:image" content="https://in.pycon.org/blog/theme/images/opengraph-default.png" />
+    {% else %}
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="{{ SITEURL }}/{{ output_file }}" />
+    <meta property="og:title" content="{{ SITENAME }}" />
+    <meta property="og:description" content="{{ DESCRIPTION }}" />
+    <meta property="og:image" content="https://in.pycon.org/blog/theme/images/opengraph-default.png" />
```

Comment:
Does using SITEURL here cause any issue?

Reply:
None that I have seen yet while testing.

Reply:
Ah I meant why why is SITEURL not used in `og:image` ?

Reply:
Open Graph tags are not used by the blog itself, but by social media websites like FB or Twitter. Websites check the `og:image` tag to find the absolute path of the image.

With the use of `SITEURL` the path is `/blog/theme/images/opengraph-default.png` which is relative and breaks the image on the preview. Hope this helps in understanding. Resolving.